packages/admin-ui/src/Radio/RadioGroupPresenter.ts

```diff
@@ -0,0 +1,48 @@
+import { makeAutoObservable } from "mobx";
+import { RadioGroupVm } from "./RadioGroup";
+import { RadioItem } from "~/Radio/RadioItem";
+import { RadioItemMapper } from "~/Radio/RadioItemMapper";
+import { RadioItemDto } from "~/Radio/RadioItemDto";
+
+interface RadioGroupPresenterParams<TValue = string> {
+    items: RadioItemDto[];
+    onValueChange: (value: TValue) => void;
+}
+
+interface IRadioGroupPresenter<TValue = string> {
+    vm: RadioGroupVm;
+    init: (props: RadioGroupPresenterParams<TValue>) => void;
+    changeValue: (value: TValue) => void;
+}
+
+class RadioGroupPresenter<TValue = string> implements IRadioGroupPresenter<TValue> {
+    private params?: RadioGroupPresenterParams<TValue> = undefined;
+
+    constructor() {
+        makeAutoObservable(this);
+    }
+
+    public init = (params: RadioGroupPresenterParams<TValue>) => {
```

Comment:
Saw this before and wanted to share that alternative to having to instantiate a class and then call `init` on it having a static `create` class, which takes params. So basically:

```ts
// Instead of this...
const rgp = new RadioGroupPresenter();
rgp.init(params);

// ...this (the `static` create method creates a new instance of the class):
const rgp = RadioGroupPresenter.create(params);
```

Maybe I missed something though, and this is probably a bit subjective. Still wanted to share, can't hurt. :)

Reply:
I'll spend some time experimenting with it; thank you for the suggestion. 😃